foundation/foundation-mda/src/main/resources/templates/deployment/jenkins/v2/jenkins.values.yaml.vm

```diff
@@ -6,5 +6,5 @@
 #aissemble-jenkins-chart:
 #  jenkins:
 #      controller:
-#        image: boozallen/${projectName}-jenkins-controller-docker
```

Comment:
S (Substantive): should `boozallen` be here?  Seems like it should not be.

Reply:
Similar comments throughout the codebase - if there were a company/agency `foo`, might this be `${dockerProjectRepositoryUrl}foo/${projectName}-jenkins-controller-docker` or just `${dockerProjectRepositoryUrl}/${projectName}-jenkins-controller-docker`?

Reply:
Dropped "boozallen" from the project specific images so they have the following format `${dockerProjectRepositoryUrl}/${projectName}-jenkins-controller-docker`. Didn't make any changes to the image names that aren't project specific.